### PR TITLE
Add release notes for R2

### PR DESCRIPTION
**Pull request by itrich**
Please add your individual release notes sections to this branch.

### CODE REVIEW

File: Release-Notes/Release2.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Release Notes for SCS Release 2
+(Release Date: 2022-03-XX)
+
+## Scope
+
+Main goals for Release 2 (R2) was ...
+
+## Component Versions
+
+* Kubernetes 1.22.2
```
Comment (garloff) [inline Release-Notes/Release2.md:10]: 1.22.3 meanwhile

File: Release-Notes/Release2.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Kubernetes Cluster API 1.0.0
```
Comment (garloff) [inline Release-Notes/Release2.md:11]: 1.0.2

Comment (garloff) [reply Release-Notes/Release2.md:11]: Should we merge this now and then have PRs against the main branch? Or PRs against a branch which gets merged to main when the release approaches?

Comment (itrich) [reply Release-Notes/Release2.md:11]: As someone might think that R2 already has been released, I would suggest working either directly on the branch or with PR against this branch.

**Comment by garloff** [comment]
Rendered doc can be read by visiting this link:
https://github.com/SovereignCloudStack/Docs/blob/feat/release-notes/Release-Notes/Release2.md

**Pull request merged**